components/esp_websocket_client/include/esp_websocket_client.h

```diff
@@ -166,6 +166,16 @@ esp_err_t esp_websocket_client_stop(esp_websocket_client_handle_t client);
  */
 esp_err_t esp_websocket_client_destroy(esp_websocket_client_handle_t client);
 
+/**
+ * @brief      If `selected` set to True, WebSocket client will destroy and free all resources at the end of event loop.
+ *
```

Comment:
We should warn users, that the client handle would be dangling and should never be used